### PR TITLE
dont interact with desktop client while in standby

### PR DESCRIPTION
**Pull request by James-Pickett**

### CODE REVIEW

File: ee/desktop/runner/runner.go [hunk 1]
```diff
@@ -531,6 +531,13 @@ func (r *DesktopUsersProcessesRunner) writeDefaultMenuTemplateFile() {
 }
 
 func (r *DesktopUsersProcessesRunner) runConsoleUserDesktop() error {
+	if r.knapsack.InModernStandby() {
```
Comment (James-Pickett) [inline ee/desktop/runner/runner.go:534]: should we skip here or in the execute loop?

Comment (RebeccaMahany) [reply ee/desktop/runner/runner.go:534]: Here makes sense! Looking here versus execute, I think when we're in modern standby we would still want to refresh menu data (https://github.com/kolide/launcher/blob/42f20a7303ca8d66ae445db673201019b4129edc/ee/desktop/runner/runner.go#L235) but maybe _not_ tell clients to pick up the new data (https://github.com/kolide/launcher/blob/42f20a7303ca8d66ae445db673201019b4129edc/ee/desktop/runner/runner.go#L439) -- so putting the `InModernStandby` check into these individual functions makes sense to me.

Comment (directionless) [reply ee/desktop/runner/runner.go:534]: We should _not_ tell the clients in standby.

I forget how menu refresh works, if we skip it, will we miss things, or will we pick things up?

Comment (James-Pickett) [reply ee/desktop/runner/runner.go:548]: we will still pick up the menu changes and can still have the most updated menu available when we come out of standby

the data to update the menu comes in via the control server update

**Review by zackattack01 — APPROVED** [review]

**Review by directionless — APPROVED** [review]
let's try!

**Pull request merged**